src/views/Predictions/Leaderboard/components/WalletStatsModal.tsx

```diff
@@ -43,8 +47,10 @@ const ExternalLink = styled(LinkExternal)`
 const WalletStatsModal: React.FC<WalletStatsModalProps> = ({ account, onDismiss, onBeforeDismiss }) => {
   const { t } = useTranslation()
   const { theme } = useTheme()
-  const result = useGetOrFetchLeaderboardAddressResult(account)
-  const profileAvatar = useGetProfileAvatar(account)
+  const selectedAddress = useGetSelectedAddress()
+  const address = account || selectedAddress
```

Comment:
This is where we accept the passed in account and default to the selected on in state.